econml/utilities.py

```diff
@@ -1636,6 +1636,7 @@ def intercept_(self):
     @property
     def _param_var(self):
         """
+
```

Comment:
Put a description here (it will be basically identical to the return) or else sphinx will complain.  Same for `_param_stderr`.

Reply:
done